addons/edi_notifier/models/email_notifier.py

```diff
@@ -38,8 +42,9 @@ def _notify(self, notifier, event_type, recs):
             if attachments:
                 email_values = {'attachment_ids': attachments.mapped('id')}
             if base_url:
-                template = template.with_context(instance_url=base_url)
+                template = template.with_context(instance_url=base_url, hostname=hostname)
             template.send_mail(rec.id, force_send=True, email_values=email_values)
+                
```

Comment:
don't think this newline is needed

Reply:
my mistake, I have removed this now also.